content/en/docs/tasks/configure-pod-container/security-context.md

```diff
@@ -149,7 +149,7 @@ exit
 
 ## Configure volume permission and ownership change policy for Pods
 
-{{< feature-state for_k8s_version="v1.18" state="alpha" >}}
+{{< feature-state for_k8s_version="v1.20" state="beta" >}}
```

Comment:
Remember to update https://kubernetes.io/docs/reference/command-line-tools-reference/feature-gates/ as well.

Reply:
Done, thanks